lib/taurus/qt/qtgui/panel/taurusform.py

```diff
@@ -380,6 +380,16 @@ def onSetFormatter(self):
                     rw.setFormat(format)
         return format
 
+    def setFormat(self, format):
```

Comment:
I think that format is a Python built-in. In this case I would suggest to use `format_`.